src/Client/AzureManaged/DurableTaskSchedulerClientOptions.cs

```diff
@@ -46,6 +46,11 @@ public class DurableTaskSchedulerClientOptions
     /// </summary>
     public bool AllowInsecureCredentials { get; set; }
 
+    /// <summary>
+    /// Gets or sets the options that determine how and when calls made to the scheduler will be retried.
+    /// </summary>
+    internal ClientRetryOptions? RetryOptions { get; set; }
```

Comment:
Any reason not to make this public?

Reply:
My original change had this as the actual gRPC retry policy, which I wanted to obscure from users with the extension methods. I forgot to remake this public after I changed to using the options you see now.